.cirrus.yml

```diff
@@ -65,7 +66,7 @@ plugin_qa_task:
     - source cirrus-env QA
     - source set_maven_build_version $BUILD_NUMBER
```

Comment:
but here you're using the one from the path, which is modified locally so you're using the correct one, but it's a bit misleading, could you also use local path, so we know a change was made, perhaps add a comment for future refactoring

Reply:
indeed, I changed to local path everywhere, however I also kept modification to the `$PATH` because `set_maven_build_version` is used from `regular_mvn_build_deploy_analyze`